datahub-web-react/src/app/entity/shared/components/styled/StatsSummary.tsx

```diff
@@ -7,31 +7,34 @@ type Props = {
 };
 
 const StatsContainer = styled.div`
```

Comment:
can we maybe add a minor padding/margin to the bottom? even like 4px or something. it looks a little tight on the bottom of the card between the stats and edge of card!